src/lvl_script.c

```diff
@@ -875,6 +878,10 @@ TbBool script_support_setup_player_as_computer_keeper(PlayerNumber plyridx, long
     return true;
 }
 
+static void null_process(struct ScriptContext *context)
```

Comment:
What's this?

Reply:
Null process used when no actions required at `runtime`